Ginger/GingerCoreNET/Run/RunSetActions/RunSetActionHTMLReportSendEmail.cs

```diff
@@ -247,28 +247,12 @@ public override void Execute(ReportInfo RI)
                     //Reporter.ToLog(eLogLevel.DEBUG, "Run set operation send Email: loggerMode is LiteDB");
                     try
                     {
-                        WorkSpace.Instance.Solution.LoggerConfigurations.SelectedDataRepositoryMethod = ExecutionLoggerConfiguration.DataRepositoryMethod.TextFile;
-                        GingerRunner gr = new GingerRunner();  // Why we create new GR here !!!!!!!!!!!!!!!
-                        runSetFolder = gr.ExecutionLoggerManager.GetRunSetLastExecutionLogFolderOffline();
-                        ReportInfo offlineReportInfo = new ReportInfo(runSetFolder);
-                        ((RunSetReport)offlineReportInfo.ReportInfoRootObject).StartTimeStamp = liteDbRunSet.StartTimeStamp;
-                        ((RunSetReport)offlineReportInfo.ReportInfoRootObject).EndTimeStamp = liteDbRunSet.EndTimeStamp;
-                        ((RunSetReport)offlineReportInfo.ReportInfoRootObject).Elapsed = liteDbRunSet.Elapsed;
-                        ((RunSetReport)offlineReportInfo.ReportInfoRootObject).RunSetExecutionRate = liteDbRunSet.ExecutionRate;
-                        ((RunSetReport)offlineReportInfo.ReportInfoRootObject).GingerRunnersPassRate = liteDbRunSet.PassRate;
-                        ((RunSetReport)offlineReportInfo.ReportInfoRootObject).EnvironmentsDetails = liteDbRunSet.Environment;
-                        CreateSummaryViewReportForEmailAction(offlineReportInfo);
+                        CreateSummaryViewReportForEmailAction(liteDbRunSet);
                         // TODO: check multi run on same machine/user
                     }
-                    finally
+                    catch(Exception ex)
                     {
-                        WorkSpace.Instance.Solution.LoggerConfigurations.SelectedDataRepositoryMethod = ExecutionLoggerConfiguration.DataRepositoryMethod.LiteDB;
                         //Reporter.ToLog(eLogLevel.DEBUG, "Run set operation send Email: Checking if runSetFolder exist: " + runSetFolder);
```

Comment:
Why not to write to Log in case of exception?
let's add it with proper Log message indicating that preparing the report mail for LiteDB is failed